.ci/docker-cpu/Dockerfile.base

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+
+ENV TZ=Europe/Berlin
+
+# Install general compilers and libraries
+RUN apt-get update -y && \
+ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone && \
+apt-get install -y software-properties-common apt-utils build-essential wget git curl && \
+apt-get install -y pkg-config cmake cmake-data tree && \
+apt-get install -y python3 python3-dev python3-pip && \
+add-apt-repository ppa:ubuntu-toolchain-r/test && \
+apt-get install -y gcc-9 g++-9 gfortran-9 && \
+update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 100 && \
+update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 100 && \
+update-alternatives --install /usr/bin/gfortran gfortran /usr/bin/gfortran-9 100 && \
+apt-get install -y unzip openmpi-bin openmpi-common libopenmpi-dev && \
+apt-get install -y hdf5-tools libhdf5-openmpi-dev libmetis-dev libparmetis-dev && \
+apt-get install -y cxxtest libarmadillo-dev libyaml-cpp-dev && \
+pip3 install --upgrade pip && \
+pip3 install numpy>=1.12.0 lxml setuptools sphinx sphinx_rtd_theme
+
+
+# Install specific libraries
+RUN wget --progress=bar:force:noscroll https://github.com/Unidata/netcdf-c/archive/refs/tags/v4.6.1.tar.gz && \
+tar -xzf v4.6.1.tar.gz && \
+cd netcdf-c-4.6.1 && CC=h5pcc ./configure --prefix=/usr --enable-shared=no --disable-dap && \
+make && make install && cd .. && rm -rf ./v4.6.1.tar.gz rm -rf ./netcdf-c-4.6.1 && \
+wget --progress=bar:force:noscroll https://github.com/libxsmm/libxsmm/archive/refs/tags/1.17.zip && \
```

Comment:
Each `RUN` statement creates a layer which will result in some space. I know that it is not tge case for my GPU image because it contains lot's of  `RUN` commands. But I needed it. In general, meany people are trying to reduce their amount of `RUN` commands